### PR TITLE
Add new key for Jetty

### PR DESCRIPTION
**Pull request by rhowe**

### CODE REVIEW

File: resources/pgp-keys-map.list [hunk 1]
```diff
@@ -877,9 +877,11 @@ org.eclipse.jgit                = 0x7C669810892CBD3148FA92995B05CCDE140C2876
 
 org.eclipse.jetty               = \
                                   0x5989BAF76217B843D66BE55B2D0E1FB8FE4B68B4, \
-                                  0x8B096546B1A8F02656B15D3B1677D141BCF3584D
+                                  0x8B096546B1A8F02656B15D3B1677D141BCF3584D, \
+											 0xB59B67FD7904984367F931800818D9D68FB67BAC
```
Comment (slawekjaranowski) [inline resources/pgp-keys-map.list:881]: please align in the same way

File: resources/pgp-keys-map.list [hunk 2]
```diff
@@ -877,9 +877,11 @@ org.eclipse.jgit                = 0x7C669810892CBD3148FA92995B05CCDE140C2876
...
 
-org.eclipse.jetty.http2         = 0x5989BAF76217B843D66BE55B2D0E1FB8FE4B68B4
+org.eclipse.jetty.http2         = 0x5989BAF76217B843D66BE55B2D0E1FB8FE4B68B4, \
+											 0xB59B67FD7904984367F931800818D9D68FB67BAC
```
Comment (slawekjaranowski) [inline resources/pgp-keys-map.list:884]: please align in the same way

**Review by slawekjaranowski — CHANGES_REQUESTED** [review]
Please provide example artifacts signed by this key

**Comment by slawekjaranowski** [comment]
Ok, I see this will fix #848

**Review by robot-simplify4u — APPROVED** [review]

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**